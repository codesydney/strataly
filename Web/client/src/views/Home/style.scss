.top-empty-div {
  height: 120px;
  background-color: #fff;
  text-align: center;
  display: flex;
  flex-direction: row;
  justify-content: center;
}
.top-empty-overlay {
  width: 100%;
  height: 100%;
  box-shadow: rgb(7, 7, 7) 0 0 0 100px inset;
  background: black;
}

.logo {
  width: 200px;
  background: transparent;
  margin: 0 20px;
}

.skyline-image {
  background: rgb(255, 255, 255);
  width: 100%;
  height: 20vh;
}
