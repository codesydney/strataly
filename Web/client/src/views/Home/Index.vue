<script>
import './style.scss'

export default {
  name: 'Home'
}
</script>

<template>
  <div class="home">
    <h1>This is the home page Woohoo!!</h1>
  </div>
</template>
