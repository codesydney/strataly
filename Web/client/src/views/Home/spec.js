import { shallowMount } from '@vue/test-utils'
import Home from './Index.vue'

describe('Home component', () => {
  it('renders test message', () => {
    const wrapper = shallowMount(Home)

    expect(wrapper.element).toMatchSnapshot()
  })
})
