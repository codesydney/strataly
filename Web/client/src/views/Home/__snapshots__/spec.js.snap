// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders test message 1`] = `
<div
  class="home"
>
  <h1>
    This is the home page Woohoo!!
  </h1>
</div>
`;
